app/react-native/src/preview/components/StoryView/index.tsx

```diff
@@ -1,19 +1,13 @@
-import React, { Component } from 'react';
-import { View, Text } from 'react-native';
+import React, {Component} from 'react';
+import {View, Text} from 'react-native';
```

Comment:
some unintended formatting changes in this file. You may need to run prettier.

Reply:
I couldn't run git hooks successfully. :(